Intersect.Editor/Forms/Editors/BaseEditorForm.cs

```diff
@@ -0,0 +1,31 @@
+using System.Windows.Forms;
+using DarkUI.Controls;
+
+namespace Intersect.Editor.Forms.Editors;
+
+public class BaseEditorForm : EditorForm
+{
+    protected DarkButton? btnSave;
+    protected DarkButton? btnCancel;
+
+    protected void SetEditorButtons(DarkButton saveButton, DarkButton cancelButton)
+    {
+        btnSave = saveButton;
+        btnCancel = cancelButton;
+    }
```

Comment:
Pick one, either the fields are protected, or you have a `Set` method, but not both.

More details in `frmQuest`